### DISCUSSION TITLE
.LAZ File Size Limit

### DISCUSSION THREAD

**Discussion by pauls99** [iTowns/itowns#1721] [Q&A]
I was trying to add a .LAZ file to the iTowns example drag-and-drop LAZ viewer, and received an OOM error in the developer console. The file was quite large (more than 750 MB), and trying again with a smaller file of approximately 2 MB worked fine. Is there a file size limit to what can be viewed?

**Comment by mgermerie** [comment]
Hi,
Thanks for the feedback. I am currently working on reproducing the OOM error when loading a large .LAZ file. I'll let you know what i find out.

**Reply by pauls99** [reply]
Hello, I wanted to check back on this question, have you been able to reproduce the OOM error? Thanks!

**Reply by gchoqueux** [reply]
I reproduce the OOM error with a large Entwine cloud.
The error is thrown when `emscripten` can't resize memory heap.

**Reply by gchoqueux** [reply]
I open a issue https://github.com/visgl/loaders.gl/issues/1912
